components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/dao/IdPManagementDAO.java

```diff
@@ -2803,6 +2803,42 @@ public IdentityProvider getIdPByRealmId(String realmId, int tenantId, String ten
 
     }
 
+    /**
```

Comment:
Add a description to the comment

Reply:
Fixed with [f32a04f](https://github.com/wso2/carbon-identity-framework/pull/4991/commits/f32a04f1db73c0d4cf20f3687a75c17ee281b6a0)